src/runtime_src/core/pcie/driver/linux/xocl/subdev/ddr_srsr.c

```diff
@@ -78,15 +76,14 @@ static int srsr_full_calibration(struct platform_device *pdev)
 	xocl_dr_reg_write32(xdev, CTRL_BIT_SYS_RST, xocl_ddr_srsr->base+REG_CTRL_OFFSET);
 	xocl_dr_reg_write32(xdev, 0x0, xocl_ddr_srsr->base+REG_CTRL_OFFSET);
 
-	for (;i<100;++i) {
+	for (; i < 100; ++i) {
```

Comment:
please define a macro for constant value like 100 for code readability.

Reply:
done, replace with FULL_CALIB_TIMEOUT 100,

full calibration should finish in 2000ms.